src/main/java/seiki/data/task/ToDo.java

```diff
@@ -0,0 +1,25 @@
+package seiki.data.task;
+
+/**
+ * Represents a todo task.
+ */
+public class ToDo extends Task {
```

Comment:
Maybe TodoTask (and similarly for event and deadline) would be class naming that implies the task inheritance

Reply:
I understand where you are coming from. Thanks!